main_test.go

```diff
@@ -27,25 +27,22 @@ import (
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v2"
 
+	"github.com/elastic/package-registry/internal/util"
 	"github.com/elastic/package-registry/packages"
-	"github.com/elastic/package-registry/util"
 )
 
 var (
 	generateFlag       = flag.Bool("generate", false, "Write golden files")
 	testCacheTime      = 1 * time.Second
 	generatedFilesPath = filepath.Join("testdata", "generated")
+	logger             = util.NewTestLogger()
```

Comment:
Suggestion: To avoid possible confusions, what about renaming this variable to something like `devLogger` or `testLogger` ? As it is defined here in "*_test.go" files, it cannot be used in the other files, but to remark that is just used here in testing.

Reply:
Done :+1: